wgpu/src/lib.rs

```diff
@@ -3192,8 +3202,18 @@ impl<'a> RenderPass<'a> {
 
     /// Draws indexed primitives using the active index buffer and the active vertex buffers.
     ///
-    /// The active index buffer can be set with [`RenderPass::set_index_buffer`], while the active
-    /// vertex buffers can be set with [`RenderPass::set_vertex_buffer`].
+    /// The active index buffer can be set with [`RenderPass::set_index_buffer`]
+    /// The active vertex buffers can be set with [`RenderPass::set_vertex_buffer`].
+    ///
+    /// Panics if indices Range is not within 0..BufferLen / BufferStride.
+    ///
+    /// indices: The index range within the index buffer.
+    /// base_vertex: Addition upon the next set of Index's. example: 0,1,2,0,2,3 on next instance
```

Comment:
```suggestion
    /// base_vertex: Added upon the next set of Index's. E.g. 0,1,2,0,2,3 on next instance
```
but don't know what you mean with "next set" here

Reply:
this was taken from jasperriz in the wgpu_user chat. Im guessing he meant that upon the next Call based on the next range of index's given that it would get added to them. Though I am still unsure how this really works fully in the backend other than what @cwfitzgerald  said in the chat of 
```rust
for instance_id in instance_range {
    for index_index in index_range {
        let vertex_id = index_buffer[index_index];
        let adjusted_vertex_id = vertex_id + base_vertex;
        let vertex = vertex[adjusted_vertex_id];
        vertex_shader(vertex, adjusted_vertex_id, instance_id);
    }
}
```

from this example I see this is only useful as a per snippet call of the function and not useful for looping as it does not Adjust the base_vertex by the base vertex on every instance or every set of index's. So yeah this needs to be fixed to remove the next set and more of a Increments the current call or something like this.

Reply:
I think it would be clearer if we just included this code in the text.

Reply:
I agree to this and we can do this!